udata/core/dataset/models.py

```diff
@@ -416,6 +459,48 @@ def update_resource(self, resource):
     def community_resources(self):
         return self.id and CommunityResource.objects.filter(dataset=self) or []
 
+    @cached_property
+    def json_ld(self):
+        result = {
+            '@context': 'http://schema.org',
+            '@type': 'Dataset',
+            '@id': str(self.id),
+            'description': mdstrip(self.description),
+            'alternateName': self.slug,
+            'dateCreated': self.created_at.isoformat(),
+            'dateModified': self.last_modified.isoformat(),
+            'url': url_for('datasets.show', dataset=self, _external=True),
+            'name': self.title,
+            'keywords': ','.join(self.tags),
+            'distribution': [resource.json_ld for resource in self.resources],
+            # This value is not standard
+            'extras': map(self.get_json_ld_extra, self.extras.items()),
```

Comment:
Pros: it removes indeed 2 character ('(' and ')').
Cons:
* It adds 14 characters
* It adds a star operator which, at least for me, makes the code harder to read

Why do you hate the parenthesis that much ;-) ?